packages/next-swc/crates/next-core/src/next_server/context.rs

```diff
@@ -279,9 +282,12 @@ pub async fn get_server_module_options_context(
         }
         ServerContextType::AppRSC { .. } => {
             let module_options_context = ModuleOptionsContext {
-                custom_ecmascript_transforms: vec![EcmascriptInputTransform::ClientDirective(
-                    StringVc::cell("server-to-client".to_string()),
-                )],
+                custom_ecmascript_transforms: vec![
+                    EcmascriptInputTransform::ClientDirective(StringVc::cell(
+                        "server-to-client".to_string(),
+                    )),
+                    EcmascriptInputTransform::ServerDirective(StringVc::cell("TODO".to_string())),
```

Comment:
Is this meant to be committed?

Reply:
yep, it's not yet supported but the API is prepared.